apps/test/unit/templates/projects/projectsReduxTest.js

```diff
@@ -54,4 +57,40 @@ describe('projectsRedux', () => {
       assert.deepEqual(nextNextState.personalProjectsList.projects.length, 3);
     });
   });
+
+  describe('startRenamingProject', () => {
+    it('startRenamingProject sets isEditing to true', () => {
+      const action = setPersonalProjectsList(stubFakePersonalProjectData);
+      const nextState = projects(initialState, action);
+      assert.deepEqual(nextState.personalProjectsList.projects, stubFakePersonalProjectData);
+      const nextAction = startRenamingProject('abcd3');
+      const nextNextState = projects(nextState, nextAction);
+      assert.deepEqual(nextNextState.personalProjectsList.projects[2].isEditing, true);
```

Comment:
Nonblocking: Here, and other places in this file where you're comparing against non-object values, `assert.deepEqual` could just be `assert.equal`.